### ISSUE TITLE
App does not start on Apple Silicon

### ISSUE THREAD

**Issue by hoegertn** [Noovolari/leapp#85]
When launching Leapp on a new MacMini M1 I get the following error.

![image](https://user-images.githubusercontent.com/1287829/109415414-e225bf00-79b8-11eb-906f-d181fdeb3471.png)

Any hints?

**Desktop (please complete the following information):**
 - OS: macOS BigSur on Apple Silicon
 - Leapp Version: 0.4.5

**Comment by andreacavagna01** [comment]
This problem is not related to the M1 silicon, this is caused because the encryption of the configuration file on Leapp is based on the Machine ID.

By changing your Mac and using Time Machine, the App is unable to decrypt the config file.

To solve it, delete the file under ~/.Leapp and the app will work.

Sorry for the inconvenience

**Comment by hoegertn** [comment]
Thanks, it worked. Maybe an error message would be great.

Does that mean the config cannot be restored from a backup?

**Comment by andreacavagna01** [comment]
exactly, for security concern of someone trying to steal the configuration encrypted file, this file can only be decrypted with the machine ID of the first system in which Leapp is installed

**Comment by hoegertn** [comment]
I think this needs to be made clear with a warning sign. People might rely on backups made by TM to store their credentials.

**Comment by andreacavagna01** [comment]
Good point, absolutely a crash of the App is not the best idea, I'll add it in the next release

**Comment by ericvilla** [comment]
We're going to add a warning statement in the [security section](https://docs.leapp.cloud/latest/security/intro/) of the docs.
